skfem/assembly/basis/boundary_facet_basis.py

```diff
@@ -101,6 +102,18 @@ def __init__(self,
                    * np.tile(self.W, (self.nelems, 1)))
         logger.info("Initializing finished.")
 
+    @classmethod
+    def init_subdomain(cls, mesh, elem, elements=None, side=0, **kwargs):
+        assert elements is not None
+        elements = mesh.normalize_elements(elements)
+        subdomain_facets, sub_counts = np.unique(mesh.t2f[:, elements],
+                                                 return_counts=True)
+        facets = subdomain_facets[sub_counts == 1]
+        tind = mesh.f2t[:, facets].flatten('F')
+        ix = np.isin(tind, elements)
+        tind = tind[ix] if side == 0 else tind[~ix]
+        return cls(mesh, elem, facets=(facets, tind, tind), **kwargs)
+
```

Comment:
Here is the constructor. Rest of the changes are less important.

Reply:
Why is it in

https://github.com/kinnala/scikit-fem/blob/85bea6cf75e196776b743b7104782530c8389d2e/skfem/assembly/basis/boundary_facet_basis.py#L20-L21

?  In ex26, for example, the interface between the "core" and "annulus" subdomains is not on the boundary of the domain; see, e.g., the snippet for extracting the interfacial facets and computing their orientation w.r.t. the two subdomains that they separate in #824 on [2021-12-29]( https://github.com/kinnala/scikit-fem/discussions/824#discussioncomment-1881795).  There a plain `FacetBasis` was used.

Reply:
Oh, I see:

https://github.com/kinnala/scikit-fem/blob/85bea6cf75e196776b743b7104782530c8389d2e/skfem/assembly/basis/__init__.py#L11

Reply:
That's a little bit confusing, and I don't think the docstring is right.